.vscode/launch.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "debug streamlit",
+            "type": "debugpy",
+            "request": "launch",
+            "module": "streamlit",
+            "args": [
+                "run",
+                "${workspaceFolder}/streamlit_template/app.py"
```

Comment:
directory paths will need to be updated here, L13, L23, L25 to match the new directory structure. `cwd` can probably be dropped since everything can just run at the root of the project now instead of it being in a sub directory.